### PR TITLE
fix scroll issue

### PR DESCRIPTION
**Pull request by cbeauchesne**

### CODE REVIEW

File: src/views/document/utils/DocumentViewMixin.js [hunk 1]
```diff
@@ -144,11 +144,34 @@ export default {
...
+                    .then(this.scrollToHash)
+                    .then(this.updateUrl)
+            }
+        },
+
+        scrollToHash() {
+            if (this.$route.hash) {
+                // we'll have to wait for DOM update
+                this.$nextTick(() => {
+                    let el = document.querySelector(this.$route.hash)
```
Comment (brunobesson) [inline src/views/document/utils/DocumentViewMixin.js]: `el` could be a const

File: src/views/document/utils/DocumentViewMixin.js [hunk 2]
```diff
@@ -144,11 +144,34 @@ export default {
...
+
+                    if (el) {
+                        let docEl = document.documentElement
```
Comment (brunobesson) [inline src/views/document/utils/DocumentViewMixin.js]: const, const and const :)

File: src/views/document/utils/DocumentViewMixin.js [hunk 3]
```diff
@@ -144,11 +144,34 @@ export default {
...
+                        let docRect = docEl.getBoundingClientRect()
+                        let elRect = el.getBoundingClientRect()
+                        let x = elRect.left - docRect.left
+                        let y = elRect.top - docRect.top
+                        window.scrollTo(x, y - 50) // navbar height ...
```
Comment (brunobesson) [inline src/views/document/utils/DocumentViewMixin.js]: I don't understand why you are computing an `x` position: you're only trying to sroll vertically to the right anchor position?

Comment (cbeauchesne) [reply src/views/document/utils/DocumentViewMixin.js]: sure :)

**Pull request merged**